docs/manual/getting_started/installation/installation.rst

```diff
@@ -0,0 +1,21 @@
+############
+Installation
+############
+
+.. |url::omg.org| raw:: html
+ 
+    <a href="https://www.omg.org/spec/DDS-PSM-Cxx/" target="_blank">DDS ISO/IEC C++ PSM API</a>
+
+The core of Cyclone DDS is implemented in C and provides C-APIs to applications. Through its C++ package, the |url::omg.org| 2003 language binding is also supported.
```

Comment:
I would make a list here of the different "extra" bindings here:
- C++
- Python
- C#

Reply:
If the core is in C, do we have any C# information?